py/aou_workbench_client/data.py

```diff
@@ -87,5 +110,13 @@ def load_data_table(cohort_name, table, columns=None, concept_ids=None,
     field_set = FieldSet(table_query)
     request = MaterializeCohortRequest(cohort_name=cohort_name,
                                        field_set=field_set,
+                                       status_filter=cohort_statuses,
                                        page_size=page_size)
-    return materialize_cohort(request, max_results=max_results, debug=debug)
+    generator = materialize_cohort(request, max_results=max_results, debug=debug)
+    if result_type == ResultTypes.GENERATOR:
```

Comment:
I didn't want to rat hole on this during the meeting, but I don't think it's worth the added complexity of a polymorphic return type to save 1/2 lines of code.

Personally I'm not convinced anyone is going to take advantage of the generator aspect unless they are familiar with the implementation details so I'd probably argue to just return a dataframe. But if you're sure this doesn't make sense as a default, perhaps add a second method like `load_data_table_df()`; should be possible to pass everything through without copy-paste via kwargs.

Reply:
OK, split this up into load_data_table (which returns a generator) and load_data_frame (which returns a DataFrame).